pkgpanda/cli.py

```diff
@@ -123,7 +123,13 @@ def run_checks(checks, install, repository):
 
 
 def main():
-    arguments = docopt(__doc__, version="Pkgpanda Package Manager")
+    arguments = docopt(
+        __doc__.format(
+            default_config_dir=constants.config_dir,
+            default_root=constants.install_root,
+            default_repository=constants.repository_base,
+        ),
```

Comment:
You could add a version string. I would suggest a semver versioning of this cli.